roles/openshift_master/tasks/openshift_from_local.yml

```diff
@@ -0,0 +1,17 @@
+- name: Copy OpenShift binary
+  copy:
+    src: openshift
+    dest: /usr/local/bin/openshift
+    mode: 0755
+
+- name: Configure OpenShift Master settings
```

Comment:
Instead of using a template here, and using lineinfile for RPM installs, why not just keep this step in main.yml and set the create parameter to yes, that way the file will be created if it doesn't exist since the lineinfile module will add the lines if the regex doesn't match.